pipeline/stages/test.yaml

```diff
@@ -130,5 +130,62 @@ stages:
                 parameters:
                   subscriptionName: ${{ variables.subscriptionName }}
                   env: ${{ variables.env }}
-            
-            
\ No newline at end of file
+
+        - ${{ if eq(variables.TURN_OFF_VM_POST_RUN, true) }}:
+          - job: 'StopVM${{parameters.env}}'
```

Comment:
Can we create this as an Azure DevOps Template in the azure-devops-templates repo please?
Can then replace this one and the Start version from above.
As the Start runs the command at the end, that will need to be moved out to another task.... hope this makes sense.

Start-Stop-Template: start=true
AzCli: Loop VM - Run Command

Start-Stop-Template: start=false

Reply:
Yeah sure. Will move it to a template.

Reply:
Moved to template in azure-devops-templates repo